consensus/state_sync/server.go

```diff
@@ -64,9 +64,7 @@ func (m *stateSync) HandleGetBlockRequest(blockReq *typesCons.GetBlockRequest) e
 	serverNodePeerAddress := consensusMod.GetNodeAddress()
 	clientPeerAddress := blockReq.PeerAddress
 
-	m.logger.Info().Fields(m.logHelper(clientPeerAddress)).Msgf("Received StateSync GetBlockRequest: %s", blockReq)
-
-	// current height is the height of the block that is being processed, so we need to subtract 1 for the last finalized block
+	m.logger.Info().Fields(m.StateSyncLogHelper(clientPeerAddress)).Msgf("Received StateSync GetBlockRequest: %s", blockReq)
```

Comment:
Are you 100% sure that using `%s` for the BlockRequest is helpful?

I'm fine with leaving it as is, as long as you were actually using it in debugging.